### ISSUE TITLE
The main Electron app topmenu window selector is only visible on macOS currently

### ISSUE THREAD

**Issue by axfelix** [jza201/frdr-secure-data#2]
Using the main `setApplicationMenu` is a good idea, but we have it hidden on non-Mac platforms right now, so users can't see it. I think this is actually something you inherited from my other codebase, it shouldn't be that way by default -- I think it's `accessWindow.setMenuBarVisibility(false);` that's doing it. I'd hidden it previously because I didn't want to expose a whole generic Cut/Copy/Undo/Reload etc. menu in my other applications, but I notice you're declaring one manually here?

Also, even if we unhide this menu, we should consider defaulting to a main page that allows people to choose their workflow; it's not always intuitive to check in the top corner menu of a single-window app. Since we have three separate workflows that all prompt for similar parameters (such as Vault credentials), I wonder if it might be a better idea to even try to support them on all with a single page, with a selector at the top, and dynamically change the contextually different fields and the functionality depending on which selector element is highlighted.

I've drawn a very quick diagram of this, will properly transcribe and digitize it a bit later.

![singlewindow](https://user-images.githubusercontent.com/252047/84176872-eaa72580-aa36-11ea-9510-e99e73d584df.png)

**Comment by axfelix** [comment]
(Also, I'm opening a bunch of issues right now just to keep TODOs straight, don't worry about getting to them all at once!)

**Comment by jza201** [comment]
A new main window is added, replacing the other threes. Also removed the application menu, which is not needed if we don't have several windows.

**Comment by axfelix** [comment]
Looks good! I think we can probably switch to using these styles for buttons too: https://getbootstrap.com/docs/4.0/components/buttons/

That way they aren't all declared as secondary and we don't have to get fussy about setting colours ourselves; we can use `btn btn-primary`, `btn btn-success`, and `btn btn-danger`.

**Comment by jza201** [comment]
Applied styles for buttons. I am using `btn-outline-secondary` instead of `btn-secondary`. When the button is clicked, the button is shown with background colour. The activate state of class `btn-secondary` is not obvious.

**Comment by axfelix** [comment]
Looks good, and I think the bootstrap default styling fits well elsewhere too! I think this can be merged for now and we can fine-tune the display later if needed, great work.

**Comment by jza201** [comment]
I forgot to mention this in my earlier comment. I did not use ajax to load content when different modes are selected. The reason is that we need to load file selector for "encrypt" and "decrypt" modes, I could not make it to work properly with ajax. I guess ajax is designed to load different content. There might be an approach, but I did not find it. So I just used a straightforward method,  adding three `div` for each extra field of each mode. When some mode is active, that `div` is shown and the other two `div` are hidden.
